packages/stryker-jasmine/src/JasmineTestFramework.ts

```diff
@@ -32,11 +32,10 @@ export default class JasmineTestFramework implements TestFramework {
   }
 
   filter(testSelections: TestSelection[]): string {
-    const ids = testSelections.map(selection => selection.id);
+    const names = testSelections.map(selection => selection.name);
```

Comment:
The randomized order does not matter for us with this change because only the id changes, right? (Since we do run non-randomized in the tests, where we use ordering)

Reply:
True